migrate-to-chimera

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+arg=$1
+if [ $# -gt 1 ] ; then
+    echo "This script get at most one argument"
+    echo "Usage:"
+    echo "$0 [-l|--logout]"
+    echo ""
+    echo "  -l | --logout   Logout after migration"
+fi
+
+if ! [ -d $HOME/.config/steam-buddy ] && \
+    ! [ -d $HOME/.local/share/steam-buddy ] && \
+    ! [ -d $HOME/.cache/steam-buddy ] ;
+then
+    # Nothing to migrate, we can exit
+    echo "Found nothing to migrate"
+    exit 0
+fi
+
+## MIGRATION MAGIC ##
+
+# Check if there are config files from steam-buddy
+if [ -d $HOME/.config/steam-buddy ] ; then
+    mv -Tbv $HOME/.config/steam-buddy $HOME/.config/chimera
+    echo "Configuration directory has been moved"
+fi
+
+# Go to our directories and replace settings:
+if [ -f $HOME/.local/share/steam-buddy/settings/settings.json ]; then
+    # Migrate settings to new location
+    echo "Moving configuration to new location"
+    if ! [ -d $HOME/.config/chimera ] ; then
+        mkdir -p $HOME/.config/chimera
+    fi
+    mv -Tbv $HOME/.local/share/steam-buddy/settings/settings.json $HOME/.config/chimera/settings.json
+    echo "Renamig relevant paths inside settings file"
+    cd $HOME/.config/chimera
+    sed -i 's/steam-buddy/chimera/g' settings.json
```

Comment:
could this potentially modify the user's password?

Reply:
Only if the user has `steam-buddy` in the username. But we can change it to match path-likes more closely.